go.mod

```diff
@@ -3,6 +3,9 @@ module altinn.studio/altinn-k8s-operator
 go 1.21
```

Comment:
We might use 1.22? (Unless there's some reason I'm not aware of). I believe it's been merged into the [latest](https://github.com/kubernetes-sigs/kubebuilder/pull/3910) version of kube builder.

Reply:
I must have forgotten to update kubebuilder CLI when I scaffolded this repo... Fixing here: https://github.com/Altinn/altinn-k8s-operator/pull/15